csharp/ql/src/Security Features/CWE-502/UnsafeDeserializationUntrustedInput.ql

```diff
@@ -24,19 +24,19 @@ where
   // intersect with strong types, but user controlled or weak types deserialization usages
   (
     exists(
-      DataFlow::Node weakTypeCreation, DataFlow::Node weakTypeUsage,
+      DataFlow::Node weakTypeUsage,
       WeakTypeCreationToUsageTrackingConfig weakTypeDeserializerTracking, MethodCall mc
     |
-      weakTypeDeserializerTracking.hasFlow(weakTypeCreation, weakTypeUsage) and
+      weakTypeDeserializerTracking.hasFlowTo(weakTypeUsage) and
```

Comment:
Could a change like this lead to a performance improvement?

Reply:
I've heard it can, since `hasFlowTo` is no longer defined in terms of `hasFlow`.